src/main/java/com/blibli/future/vo/ReportVo.java

```diff
@@ -0,0 +1,243 @@
+package com.blibli.future.vo;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ReportVo implements Serializable{
+	private static final long serialVersionUID = 2006346112358122493L;
```

Comment:
Is this a response VO? I think it will be easier to identify between request VO and response Vo if this class is renamed to `ReportResponseVO`.
I'll change the other response class if i happen to meet one